src/gui/tray/activitylistmodel.cpp

```diff
@@ -730,6 +716,46 @@ void ActivityListModel::triggerCaseClashAction(Activity activity)
     ownCloudGui::raiseDialog(_currentCaseClashFilenameDialog);
 }
 
+void ActivityListModel::displaySingleConflictDialog(const Activity &activity)
+{
+    Q_ASSERT(!activity._file.isEmpty());
+    Q_ASSERT(!activity._folder.isEmpty());
+    Q_ASSERT(Utility::isConflictFile(activity._file));
+
+    const auto folder = FolderMan::instance()->folder(activity._folder);
+
+    const auto conflictedRelativePath = activity._file;
+    const auto baseRelativePath = folder->journalDb()->conflictFileBaseName(conflictedRelativePath.toUtf8());
+
+    const auto dir = QDir(folder->path());
+    const auto conflictedPath = dir.filePath(conflictedRelativePath);
+    const auto basePath = dir.filePath(baseRelativePath);
+
+    const auto baseName = QFileInfo(basePath).fileName();
+
+    if (!_currentConflictDialog.isNull()) {
+        _currentConflictDialog->close();
+    }
+    _currentConflictDialog = new ConflictDialog;
```

Comment:
@mgallien I am wondering, what will happen with a previous instance of `ConflictDialog`? Will it get destroyed when assigned to a `QPointer`?

Reply:
no idea
existing code
I think this works fine